packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -73,15 +58,14 @@ function ListViewBlockSelectButton(
 				) }
 				onClick={ onClick }
 				onKeyDown={ onKeyDownHandler }
-				aria-describedby={ descriptionId }
 				ref={ ref }
 				tabIndex={ tabIndex }
 				onFocus={ onFocus }
 				onDragStart={ onDragStartHandler }
 				onDragEnd={ onDragEnd }
 				draggable={ draggable }
 				href={ `#block-${ clientId }` }
-				aria-expanded={ isExpanded }
+				aria-hidden={ true }
```

Comment:
@andrewserong What if you only added `aria-hidden="true"` during selection? The issue is, it no longer announces a link to screen readers so users may not understand this block title is clickable. Worth a test?

Reply:
From my explorations yesterday, in order to remove the duplicate announcements of the block, I think we need to permanently set `aria-hidden` to `true` at the button level. Otherwise the cell reads out the label from the child, and then the link is announced as well. If we'd like to better highlight that it's a link, I wonder if we can update the `label` that we're setting on the cell? At the moment I've just used the block title, but we could make this anything we like.

Reply:
@andrewserong Can you add the word "link"?

Thanks.